ginisdk/src/doc/source/guides/getting-started.rst

```diff
@@ -74,7 +74,24 @@ with this configuration would be ``550e8400-e29b-11d4-a716-446655440000@example.
     DocumentTaskManager documentManager = gini.getDocumentTaskManager();
 
 
-Congratulations, you successfully integrated the Gini SDK. 
+Congratulations, you successfully integrated the Gini SDK.
+
+Using the Gini Accounting API
+=============================
+
+In version 2.3.0 we added support for the Gini Accounting API. To use it simply set the Gini API type
+to `GiniApiType.ACCOUNTING` when using the `SdkBuilder`:
+
+.. code-block:: java
+
+    Gini gini = new SdkBuilder(getContext(), "gini-client-id", "GiniClientSecret", "example.com")
```

Comment:
Why the client has dashes and the secret is camelCased? Just curious 😛

Reply:
I found who we should ask:<img width="1309" alt="screenshot 2019-01-16 at 17 08 56" src="https://user-images.githubusercontent.com/13149498/51261903-7edf9b00-19b1-11e9-8ef9-f760eeb92b1f.png">